src/apps/testapps/testKRing.c

```diff
@@ -24,6 +24,52 @@
 #include "baseCells.h"
 #include "h3Index.h"
 #include "test.h"
+#include "utility.h"
+
+void kRing_equals_kRingInternal_assertions(H3Index h3) {
+    for (int k = 0; k < 3; k++) {
+        int kSz = H3_EXPORT(maxKringSize)(k);
+
+        H3Index *neighbors = calloc(kSz, sizeof(H3Index));
+        int *distances = calloc(kSz, sizeof(int));
```

Comment:
Looks like stack allocations make more sense, unless this is an example to avoid difficulties with build tools.

Reply:
This was refactored out of a previous test without updating the allocations.